ddtrace/debugging/_capture/utils.py

```diff
@@ -140,14 +142,24 @@ def capture_exc_info(exc_info):
     }
 
 
-def capture_value(value, level=MAXLEVEL, maxlen=MAXLEN, maxsize=MAXSIZE, maxfields=MAXFIELDS):
-    # type: (Any, int, int, int, int) -> Dict[str, Any]
+def capture_value(value, level=MAXLEVEL, maxlen=MAXLEN, maxsize=MAXSIZE, maxfields=MAXFIELDS, hg=None):
+    # type: (Any, int, int, int, int, Optional[HourGlass]) -> Dict[str, Any]
+    def time_permits(_=None):
```

Comment:
why do we need '_=None' ?

Reply:
so that we can call it with arguments (for the `takewhile`) and without (for the other usages in the code).